monitoring/heartbeats/in-servicepulse.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Managing endpoint heartbeats in ServicePulse
+summary: Describes how ServicePulse monitors endpoints activity and availability using heartbeat messages
+reviewed: 2018-01-05
+component: Heartbeats
+versions: 'Heartbeats:*'
+---
+
+ServicePulse relies on heartbeat messages sent from the heartbeat plugin to indicate whether an endpoint is active or inactive. The main dashboard shows a heartbeats icon which will indicate if there are any inactive endpoints.
+
```

Comment:
Add note that heartsbeats only indicate if an instance is running, not if it is correctly processing messages?

Reply:
I'm not sure this belongs here. It is already in the main heartbeats index page

Reply:
True, the order I reviewed  is not the order of the content.